libs/core/langchain_core/runnables/base.py

```diff
@@ -348,6 +348,7 @@ def get_input_schema(
         return create_model(
             self.get_name("Input"),
             __root__=root_type,
+            __module_name=self.__class__.__module__,
```

Comment:
some comments would help, dont have an intuition for why this is needed

Reply:
added comments